src/Microsoft.Framework.Logging/LoggerFactory.cs

```diff
@@ -10,11 +10,12 @@ namespace Microsoft.Framework.Logging
     /// <summary>
     /// Summary description for LoggerFactory
     /// </summary>
-    public class LoggerFactory : ILoggerFactory
+    public class LoggerFactory : ILoggerFactory, IDisposable
```

Comment:
The dispose on this factory should be called in Hosting...do you want us to take care of it?
https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNet.Hosting/WebHostBuilder.cs#L53